### ISSUE TITLE
Add Introduction section to docs 

### ISSUE THREAD

**Issue by tbrowder** [Raku/doc#4526]
## Add an Introduction "section"

The current online docs need a short introductory para introducing the page.Include a link to the online version of the ‘Raku.guide’ [https:/raku.guide].

**Comment by tbrowder** [comment]
My bad. Closing.

**Comment by coke** [comment]
No, this one is fine, we can use better text on the intro page!
